addon/adapters/rest.js

```diff
@@ -853,7 +853,7 @@ export default Adapter.extend(BuildURLMixin, {
           requestData
         );
 
-        if (response instanceof AdapterError) {
+        if (response.isAdapterError) {
```

Comment:
Should likely be `response && response.isAdapterError` in case `response` is undefined / null.